specs/latest/2.0/index.html

```diff
@@ -699,12 +694,23 @@ <h2 class="no-toc">Table of contents</h2>
   const GLenum MAX_CLIENT_WAIT_TIMEOUT_WEBGL                 = 0x9247;
 
   /* Buffer objects */
+  void bufferData(GLenum target, BufferDataSource? srcData, GLenum usage,
+                  optional unsigned long srcOffset = 0,
+                  optional unsigned long length = 0);
+  void bufferSubData(GLenum target, GLintptr dstByteOffset,
+                     BufferDataSource srcData,
+                     optional unsigned long srcOffset = 0,
+                     optional unsigned long length = 0);
+
```

Comment:
Then we should modify the WebGL1 spec also.  Otherwise we will have to branch test code path with WebGL1 and WebGL2 behaviors on these entry points. Also, won't there be a function overload conflicts if WebGL1 and WebGL2 differ?